procd: update to git HEAD

robimarko opened the pull request:
```
ca8c302 inittab: fallback when multiple "console=" is detected
49ea930 utils: add key-value offset support to get_cmdline_val()
```

Required for the recent Elecom multiple console commits.

musashino205 commented:
> inittab: fallback when multiple "console=" is detected

a short hash may be missing

Ansuel commented:
@robimarko use the maintainer update_git_source_package.sh

robimarko committed:
procd: update to Git HEAD (2024-03-25)

79f8cfa58ee7 ci: add github test workflow
428f40e7984f test commit fixing warnings
63058d1f81a5 ci: enable ujail builds
49ea930a862c utils: add key-value offset support to get_cmdline_val()
ca8c30208d5e inittab: fallback when multiple "console=" is detected

Required for the recent Elecom multiple console commits.

Signed-off-by: Robert Marko <robimarko@gmail.com>

robimarko commented:
@Ansuel Thanks for pointing at that script, used it now.
